ibm/service/cis/data_source_ibm_cis_domain.go

```diff
@@ -28,6 +28,12 @@ func DataSourceIBMCISDomain() *schema.Resource {
 				Description: "CISzone - Domain",
 				Required:    true,
 			},
+			cisDomainType: {
```

Comment:
In datasource its a conputed not optional with default

Reply:
It should be Optional with default. It is not computed.

Reply:
Corrected this